pkgs/top-level/python-packages.nix

```diff
@@ -3129,6 +3129,8 @@ self: super: with self; {
 
   duckduckgo-search = callPackage ../development/python-modules/duckduckgo-search { };
 
+  duct-py = callPackage ../development/python-modules/duct-py { };
```

Comment:
Read the [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md) guide **before** marking the "_Fits CONTRIBUTING.md_" checkbox.

> 
>   Format the commit messages in the following way:
> 
>     (pkg-name | nixos/<module>): (from -> to | init at version | refactor | etc)
> 
>     (Motivation for change. Link to release notes. Additional information.)
> 
> For consistency, there should not be a period at the end of the commit message's summary line (the first line of the commit message).
> 
> Examples:
> 
>     nginx: init at 2.0.1